### PR TITLE
Se creo el modal para el perfil del usuario

### PR DESCRIPTION
**Pull request by JoseJuanM**
Solo falta el PUT al editar la informacion del usuario

### CODE REVIEW

**Comment by 0aps** [comment]
@JoseJuanM couple of items to check:

1. Write the commit message in english and follow [these guidelines](https://cbea.ms/git-commit/).
2. Fix merge conflicts

File: microservices/frontend/src/components/app/App.js [hunk 1]
```diff
@@ -12,9 +12,17 @@ import { ConfirmationPage } from '../../modules/login/ConfirmationPage';
...
 import { RestorePage } from '../../modules/login/RestorePage';
 import { ToastContainer } from 'react-toastify';
+import { useState } from 'react';
+import React from 'react';
 
 export function App () {
   const { loading, user } = useStoreWithInitializer(({ app }) => app, load);
+  const [parentCallBack, setParentCallBack] = useState(false)
+
+  const onTrigger = (event) => {
```
Comment (0aps) [inline microservices/frontend/src/components/app/App.js]: there's no need to do this on the App component. We should handle this as part of the App reducer. (the state to display or not the modal)

File: microservices/frontend/src/components/navbar/NavBar.js [hunk 2]
```diff
@@ -50,16 +51,18 @@ function GuestLinks () {
...
   return (
     <>
       <NavItem>
-        <div className='profile-picture'>
-          <img
-            className='pp-user'
-            src='images/profile_placeholder.svg' alt='profile'
-          />
-        </div>
+        <button name="myname" onClick={onTrigger}>
```
Comment (0aps) [inline microservices/frontend/src/components/navbar/NavBar.js]: We should remove this and handle it directly on the profile component.

File: microservices/frontend/src/components/profile-modal/ProfileModal.js [hunk 3]
```diff
@@ -1,3 +1,186 @@
...
+`;
+
+const ButtonsContainer = styled.div`
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  margin-top: 15px;  
+`;
+
+const Contenido = styled.div`
```
Comment (0aps) [inline microservices/frontend/src/components/profile-modal/ProfileModal.js]: Change the name of the component to english

**Review by 0aps — CHANGES_REQUESTED** [review]

**Review by JoseJuanM — COMMENTED** [review]
Ok, I will make the changes

**Comment by 0aps** [comment]
Please resolve merge conflicts on your branch (i.e pull from main on this repository) @JoseJuanM

File: microservices/frontend/src/components/app/App.js [hunk 4]
```diff
@@ -26,7 +26,7 @@ export function App () {
       {loaded && (
         <div className='d-flex flex-column min-vh-100'>
           <ToastContainer position='bottom-right' />
-          <Header user={user} logout={logout} />
+          <Header user={user} logout={logout} showModal={showModal} handleOpen={handleOpen} handleClose={handleClose}/>
```
Comment (0aps) [inline microservices/frontend/src/components/app/App.js:29]: We probably can improve here but let's merge and solve it later.

**Pull request merged**